### DISCUSSION TITLE
EU (Europe, inc. UK) Z-Wave Long Range (Z-Wave LR) controller firmware for all Z-Wave 800 Series in the European market (EULR RF Region, i.e. 868.42 MHz)?

### DISCUSSION THREAD

**Discussion by Hedda** [zwave-js/zwave-js#7257] [General]
Any news on EU/UK Long Range (ZWLR) Controller firmware for Z-Wave 800 Series for the European market (868.42 MHz RF Region EULR)?

* https://z-wavealliance.org/news_p/z-wave-alliance-announces-release-of-z-wave-long-range-specification-for-european-market/
  * https://z-wavealliance.org/what-is-z-wave-long-range-how-does-it-differ-from-z-wave/
    * https://www.silabs.com/wireless/z-wave/z-wave-long-range-overview

For reference,  Z-Wave Long Range ( Z-Wave LR) support for EU-LR (Europe and United Kingdom) was recently added to Z-Wave JS v13.3.0

* https://github.com/zwave-js/node-zwave-js/releases/tag/v13.3.0

On the Silicon Labs SDK and firmware side they recently added initial ZWLR Europe region firmware support in experimental (Alpha quality) as first introduced in Z-Wave and Z-Wave Long Range 800 SDK 7.22.0 GA that is part of Simplicity SDK Suite 2024.6.0 released June 5, 2024.

* https://www.silabs.com/documents/public/release-notes/SRN14930-7.22.0.0.pdf

@AlCalzone mention in https://github.com/zwave-js/node-zwave-js/pull/6751 that ZWLR for EU (EULR) can now be tested on Silabs official [Z-Wave 800 Series Development Kit](https://www.silabs.com/development-tools/wireless/z-wave/z-wave-800-dev-kit) using Silabs Simplicity Studio and new Simplicity SDK from Silicon Labs in version 7.22.1 firmware builds (and presumably later firmware versions as well).

* https://github.com/zwave-js/node-zwave-js/pull/6751

That should now work (experimentally) with controllers based on EFR32ZG23 (ZG23) EFR32ZG23 (ZG23) chip or ZGM230S module:

* https://github.com/zwave-js/node-zwave-js/discussions/3409

He also mentions there that the latest firmware versions of Z-Wave.me hardware should in theory  also support EU LR, but think they are 700 series, but understod it as Long Range for Europe will only be supported on Z-Wave 800 Series radio chips from Silabs, or is that not the case?

* https://github.com/zwave-js/node-zwave-js/discussions/4112

I understand Zooz currently only provides Long Range compatible firmware for the  US/CA/MX (United States, Canada, Mexico) variants of their Zooz ZST39 Zwave Controller and their Zooz 800 Series Z-Wave Long Range GPIO Module ZAC93 LR:

* https://github.com/zwave-js/node-zwave-js/discussions/5615

    * https://www.thesmartesthouse.com/products/zooz-800-series-z-wave-long-range-usb-stick-zst39
    * https://www.thesmartesthouse.com/products/zooz-800-series-z-wave-long-range-gpio-module-zac93-lr

Is it enough to with that ZST39 800 Z-Wave Stick Firmware 1.50 (based on SDK 7.22.1) if you have an EU/UK variant of Zooz ZST39 or ZAC93?

* https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/
 *  https://www.support.getzooz.com/kb/article/1352-zst39-800-long-range-z-wave-stick-change-log/
    * https://www.support.getzooz.com/kb/article/1425-zst39-long-range-usb-stick-faqs/
    * https://www.support.getzooz.com/kb/article/1276-how-to-perform-an-otw-firmware-update-on-your-zst39-800-long-range-z-wave-stick/

PS: So what are the benefits of Z-Wave Long Range (also marketed as "Z-Wave Plus LR" and "Z-Wave LR")?
  1. One of the most exciting benefits is the massive increase in transmission range, done by supports a maximum output power of 30dBm.
  2. Improved battery life by leveraging dynamic power control via auto-adjusting and optimizing radio output power at every transmission. 
  3. ZWLR is also capable of supporting up to 4000 nodes on a single network by increasing the addressing space to 12-bit!
  4. ZWLR will allow you to keep your mesh network for non-LR devices and connect real LR devices directly to the controller point-to-point,

![image](https://github.com/user-attachments/assets/9edad325-1927-4de1-b75f-6a5431d80340)

![image](https://github.com/user-attachments/assets/42afdb25-50a2-4063-941b-c9f7c5952266)

**Comment by AlCalzone** [comment]
AFAIK the plan is to release EU_LR end of 2024, so we should expect to see the first devices next year. Testing can be done with development kits and any 800 series controller based on firmware 7.22.1+
Z-Wave.me controllers on current firmwares also work despite them being 700 series, because they managed to include the EU_LR support somehow.

**Reply by mundschenk-at** [reply]
Do you have any info on this? I have been in contact with a vendor and while they were excited about EU_LR last year, that seems to have completely stalled.

**Reply by kpine** [reply]
As of https://github.com/SiliconLabs/simplicity_sdk/releases/tag/v2024.12.0 and Z-Wave SDK 7.23.0.0:

> Z-Wave Long Range European region is officially supported

**Reply by Hedda** [reply]
There will obviously be a while before we get Z-Wave LR certified devices available in the European Union, but really great so see that we at least can have Z-Wave Long Range firmware in the EU for all Z-Wave 800 Series ZWave Controller now.

**Reply by AlCalzone** [reply]
If I understand it correctly, EU_LR will land in the 2025A certification program, which means EU_LR devices can be certified starting mid 2025.

**Comment by Esmax666** [comment]
@Hedda 
thx for the news :)
Currently i have the Zwave 500


i guess it s better end of 2024 in order to have the new FW ?
What do you recommand to take as 800 Conroler ?
this one ? 
https://www.amazon.de/ZWave-Stick-Z-Wave-Verwendung-Assistant/dp/B0CFVZBGMH?th=1
or this one ?
https://www.domadoo.fr/fr/box-domotique/7041-zooz-controleur-usb-z-wave-800-853478006391.html

**Reply by Hedda** [reply]
> i guess it s better end of 2024 in order to have the new FW ?
> What do you recommand to take as 800 Conroler ?

Firmware is not mature and migration from Z-Wave 300/500/700 to Z-Wave 800 is still far from easy, however, I read that Z-Wave JS developers are working on streamlining the migration process, so unless you are a developer then I would actually recommend that you instead just wait until the Nabu Casa team releases their upcoming official Home Assistant Z-Wave Controller that Z-Wave JS will have the best out-of-the-box support for (in hopefully by the end of 2024) which will not only make maintaining backups and firmware a breeze but also look to have an external antenna that could help even more with reception from devices that are not long-range products:

* https://www.home-assistant.io/blog/2024/05/08/zwave-is-not-dead/
  * https://www.home-assistant.io/blog/2024/05/08/zwave-is-not-dead/#range-testing-our-z-wave-stick-prototype
* https://www.home-assistant.io/blog/2024/09/24/we-are-joining-the-z-wave-alliance-board/
* https://www.home-assistant.io/blog/2024/06/12/roadmap-2024h1
  * https://www.home-assistant.io/blog/2024/06/12/roadmap-2024h1#z-wave

PS: I've however ordered Zooz ZST39 directly from thesmartesthouse.com (which is the team behind Zooz) to use in my own lab/test setup (I made a co-order for 4 of them with four friends so we split the shipping cost and import + custom fees as was a better deal): 

* https://www.getzooz.com/zooz-zst39-z-wave-long-range-usb-stick/
  * https://www.thesmartesthouse.com/collections/controllers/products/zooz-800-series-z-wave-long-range-usb-stick-zst39?variant=40054649487423

**Reply by mundschenk-at** [reply]
What issue do you see for migration from 500 to 800, unless you have a very old SDK on the 500 series stick? I didn't experience any problems restoring the NVM backup from my old Aeotec Gen5+ to a Zooz ZST39.

**Reply by Hedda** [reply]
> Firmware is not mature and migration from Z-Wave 300/500/700 to Z-Wave 800 is still far from easy, however, I read that Z-Wave JS developers are working on streamlining the migration process

See relevant developer ideas here about making migration from legacy controllers easier when NVM backup/restore is not an option: 

* https://github.com/zwave-js/node-zwave-js/issues/7096

The point is that migrations are not always straight-forward or user-friendly so user experience needs to be made more streamlined.

Also note there is no NVM backup/restore feature built-into Home Assistant's Z-Wave JS integration when not using Z-Wave JS UI:

* https://www.home-assistant.io/integrations/zwave_js/

Home Assistant recommend its users to use the built-in Z-Wave JS add-on in Home Assistant (not a seperate Z-Wave JS server):

* https://www.home-assistant.io/integrations/zwave_js/#setting-up-a-z-wave-js-server

PS: Home Assistant has today almost 100,000 Z-Wave JS users (estimated as only an estimated third choose to opt in for Analytics):

* https://analytics.home-assistant.io/integrations/
   * https://analytics.home-assistant.io/
     * https://www.home-assistant.io/integrations/analytics

**Reply by Esmax666** [reply]
ok thanks a lot, then a will wait until the Problem are solved
This update will be running also for HomeAssitant And also Iobroker ?

Can we see if the percentage of ZWave users is increasing in the stats you found (I couldn't find them)?
I wanted to see which communication protocol is becoming the favorite :)

Zwave 9,2%: https://www.home-assistant.io/integrations/zwave_js/#setting-up-a-z-wave-js-server
zigbee 25%; https://www.home-assistant.io/integrations/zha
MEater: 17,7 https://www.home-assistant.io/integrations/matter

Thanks

**Reply by Hedda** [reply]
> Can we see if the percentage of ZWave users is increasing in the stats you found (I couldn't find them)?

Not that webpage, it only tracks active "opt in" installations at https://analytics.home-assistant.io/

> I wanted to see which communication protocol is becoming the favorite :)

You can/could track current users yourself on https://analytics.home-assistant.io/integrations/
